akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorContextAdapter.scala

```diff
@@ -110,6 +110,7 @@ private[akka] object ActorContextAdapter {
   }
   override private[akka] def internalSpawnMessageAdapter[U](f: U => T, _name: String): ActorRef[U] = {
     val cell = classicContext.asInstanceOf[akka.actor.ActorCell]
+    import akka.actor.actorRef2Scala
```

Comment:
so this is now
https://dotty.epfl.ch/docs/reference/dropped-features/package-objects.html